src/routes/cards/[cardsName]/+page.ts

```diff
@@ -0,0 +1,32 @@
+  },
+  [CardNames.Thomas]: {
+    passGoogle:
+      'https://pay.google.com/gp/v/save/eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJidXNpbmVzcy1jYXJkc0BidXNpbmVzcy1jYXJkcy0zOTk2MDkuaWFtLmdzZXJ2aWNlYWNjb3VudC5jb20iLCJhdWQiOiJnb29nbGUiLCJvcmlnaW5zIjpbInd3dy5leGFtcGxlLmNvbSJdLCJ0eXAiOiJzYXZldG93YWxsZXQiLCJwYXlsb2FkIjp7ImdlbmVyaWNDbGFzc2VzIjpbeyJpZCI6IjMzODgwMDAwMDAwMjIyNjgwMjAudGhvbWFzLXZlcm1vdXRoMSJ9XSwiZ2VuZXJpY09iamVjdHMiOlt7ImlkIjoiMzM4ODAwMDAwMDAyMjI2ODAyMC50aG9tYXMtdmVybW91dGgiLCJjbGFzc0lkIjoiMzM4ODAwMDAwMDAyMjI2ODAyMC50aG9tYXMtdmVybW91dGgxIiwic3RhdGUiOiJBQ1RJVkUiLCJsb2dvIjp7InNvdXJjZVVyaSI6eyJ1cmkiOiJodHRwczovL2ltYWdlZGVsaXZlcnkubmV0L3JPVGM5dEtDVFFCYzl6dGtpQlRYX3cvZjczMDE2MGItYTc5ZC00ZGYzLTcwNmEtNjY3YTEwOWRhNTAwL3B1YmxpYyJ9LCJjb250ZW50RGVzY3JpcHRpb24iOnsiZGVmYXVsdFZhbHVlIjp7Imxhbmd1YWdlIjoiZGEtREsiLCJ2YWx1ZSI6IkxPR09fSU1BR0VfREVTQ1JJUFRJT04ifX19LCJjYXJkVGl0bGUiOnsiZGVmYXVsdFZhbHVlIjp7Imxhbmd1YWdlIjoiZGEtREsiLCJ2YWx1ZSI6IlZlcm1vdXRoLm51In19LCJoZWFkZXIiOnsiZGVmYXVsdFZhbHVlIjp7Imxhbmd1YWdlIjoiZGEtREsiLCJ2YWx1ZSI6IlRob21hcyBWYXNlIn19LCJ0ZXh0TW9kdWxlc0RhdGEiOlt7ImlkIjoidGVsOiIsImhlYWRlciI6IlRlbDoiLCJib2R5IjoiKzQ1IDI3IDUxIDEzIDQ5In0seyJpZCI6Im1haWw6IiwiaGVhZGVyIjoiTWFpbDoiLCJib2R5IjoiaW5mb0B2ZXJtb3V0aC5udSJ9LHsiaWQiOiJjdnIiLCJoZWFkZXIiOiJDVlI6IiwiYm9keSI6IjQwNjQ5Mzk1In0seyJpZCI6ImFkZHJlc3MiLCJoZWFkZXIiOiJBZGRyZXNzIiwiYm9keSI6ImMvbyBUaG9tYXMgVmFzZSBCb2xzZXRoIFxuT3ZlcmdhZGVuIE5lZGVuIFZhbmRldCA0OSBCLCBzdCAudHYgXG4xNDE0IEvDuGJlbmhhdm4gSyJ9XSwiaGV4QmFja2dyb3VuZENvbG9yIjoiI2YwYWFhOSIsImJhcmNvZGUiOnsidHlwZSI6IlFSX0NPREUiLCJ2YWx1ZSI6Imh0dHBzOi8vd3d3LnZlcm1vdXRoLm51L2NhcmRzL3Rob21hcyIsImFsdGVybmF0ZVRleHQiOiJTaGFyZSJ9LCJoZXJvSW1hZ2UiOnsic291cmNlVXJpIjp7InVyaSI6Imh0dHBzOi8vaW1hZ2VkZWxpdmVyeS5uZXQvck9UYzl0S0NUUUJjOXp0a2lCVFhfdy85MDRhYzEyNC1kZDAxLTQ0NWEtMzI2ZC0xYTEzZTM2MjUyMDAvcHVibGljIn0sImNvbnRlbnREZXNjcmlwdGlvbiI6eyJkZWZhdWx0VmFsdWUiOnsibGFuZ3VhZ2UiOiJkYS1ESyIsInZhbHVlIjoiSEVST19JTUFHRV9ERVNDUklQVElPTiJ9fX19XX0sImlhdCI6MTY5NjI1MjU3MX0.hdPNaQlKmhCNWEHchl_F64-oo-zSfI4oaaFslea3mjZcKuiJKyP5WJJcEhQ6e69ioydJgLImiYDRscpUl5Zh16gIOdGs6veAcz4TWc7R9RixRNMDeTvdl0-8oP3HOilMoWVEYEe0ylQ5E9PWhgKkHtjb8um4Tjt6UKdL9Ban4uoVNMUCMxNVzeU268RiLD61kKEF-Ejul7ZW7-sC4Zx7xmvTjl6DhEEpNw_zI5uiXNY71PEPOd_vhtHA0rJx-yGwW6etLhwDv1uiCqg98wVhGo9bulq4zD5KqywC1wr6TotKD2mw8N-HVGpHs-dwYQ9x-T7ZNpV37LSqfJz-FDUUmA',
```

Comment:
why the extra space?

Reply:
There is no extra space, only the one that exists to separate the cost from the function.